web/modules/custom/dpl_favorites_list/src/Form/FavoritesListSettingsForm.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\dpl_favorites_list\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Favorites list setting form.
+ */
+class FavoritesListSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames(): array {
+    return [
+      'favorites_list.settings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'favorites_list_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    $config = $this->config('favorites_list.settings');
+
+    $form['settings'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Basic settings'),
+      '#tree' => FALSE,
+    ];
+
+    $form['settings']['page_size_mobile'] = [
+      '#type' => 'number',
+      '#title' => $this->t('Page size mobile'),
+      '#default_value' => $config->get('page_size_mobile') ?? 25,
```

Comment:
Not fond of hardcoding values on the fly. At least it should be possible to export the config and make it a part of this PR.

Reply:
This is currently following the same pattern as in the other PR's that have been approved and merged.

Reply:
I understand that you are under time pressure and choose to ignore my request.